templates_jinja2/event_partials/event_table.html

```diff
@@ -42,7 +42,7 @@
             {% endif %}
           {% endif %}
         </td>
-        <td>{% if event.start_date %}<time itemprop="startDate" datetime="{{event.start_date|strftime("%c")}}">{{ event.start_date|strftime("%b %d") }}{% if event.start_date.date() != event.end_date.date() %}</time> to <time itemprop="endDate" datetime="{{event.end_date|strftime("%c")}}">{{ event.end_date|strftime("%b %d") }}{% endif %}{{ event.end_date|strftime(", %Y") }}</time>{% endif %}</td>
```

Comment:
`strftime("%c")` actually isn't what we're looking for here - my guess is this got botched from a `date:c` conversion. The `c` format is isoformat https://github.com/django/django/blob/master/django/utils/dateformat.py#L217

Reply:
:+1: